src/entity/Source.ts

```diff
@@ -165,4 +165,7 @@ export class SquadSource extends Source {
 
   @Column({ default: 0 })
   memberInviteRank: number;
+
+  @Column({ default: false })
```

Comment:
Missing this part of RFC in regarding default value when squad is created? https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1151500290/RFC-069+-+Admin+approval+on+Squads+posts#Squad-creation